src/main/java/com/conveyal/datatools/manager/models/Label.java

```diff
@@ -0,0 +1,89 @@
+package com.conveyal.datatools.manager.models;
+
+import com.conveyal.datatools.manager.auth.Auth0UserProfile;
+import com.conveyal.datatools.manager.persistence.Persistence;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.mongodb.client.model.Filters.eq;
+
+public class Label extends Model implements Cloneable{
+    public static final Logger LOG = LoggerFactory.getLogger(Label.class);
+
+    /**
+     * The collection of which this Label is a part
+     */
+    public String projectId;
+
+    /**
+     * Get the Project of which this label is a part
+     */
+    public Project retrieveProject() {
+        return projectId != null ? Persistence.projects.getById(projectId) : null;
+    }
+
+    @JsonProperty("organizationId")
+    @JsonIgnore
+    public String organizationId () {
+        Project project = retrieveProject();
+        return project == null ? null : project.organizationId;
+    }
```

Comment:
Seems like the `retrieveProject` method contents could be inlined in here since that appears to be the only place it is used.

Reply:
Inlined in 9ac824a